### ISSUE TITLE
FileOpenPicker force deactivate viewmodel

### ISSUE THREAD

**Issue by xlazom00** [Caliburn-Micro/Caliburn.Micro#120]
FileOpenPicker is new api from WP silverlight 8.1  when you want to select some files or pictures
it has 2 phases
1.select source or provider
2. select content (pciture, video or files)

but If I press BackButton on 1. phase, my actual view model get Deactive.
I think this is bug in caliburn
If I press backbutton on 2.phase, everything go fine

Can Anybody look at this
thx

I made example
http://1drv.ms/1C6WGYW

**Comment by nigel-sampson** [comment]
I've managed to recreate it, but not yet sure of the solution.

The first step in the file picker shows up as a `Flyout`, you're technically still in your app which is why your view model isn't deactivated. Normally when a flyout is open like this it intercepts any back button presses and cancels them so we don't see anything like that.

It appears that the flyout isn't intercepting the back button causing your app to navigate back from PivotPage to MainPage.

It's not a bug in Caliburn.Micro as I've managed to recreate this in a pure app.

Still looking at possible solutions.
